pkgs/development/tools/build-managers/bear/default.nix

```diff
@@ -40,11 +52,29 @@ stdenv.mkDerivation rec {
     re2
   ];
 
+  cmakeFlags = [
+    # Build system and generated files concatenate install prefix and
+    # CMAKE_INSTALL_{BIN,LIB}DIR, which breaks if these are absolute paths.
+    "-DCMAKE_INSTALL_BINDIR=bin"
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+  ];
+
   patches = [
-    # Default libexec would be set to /nix/store/*-bear//nix/store/*-bear/libexec/...
-    ./no-double-relative.patch
+    # Fix toolchain environment variable handling and the Darwin SIP check.
+    ./fix-functional-tests.patch
   ];
 
+  postPatch = ''
+    patchShebangs test/bin
+
+    # /usr/bin/env is used in test commands and embedded scripts.
+    find test -name '*.sh' \
+      -exec sed -ie 's|/usr/bin/env|${coreutils}/bin/env|g' {} +
+  '';
+
+  # Functional tests use loopback networking.
+  __darwinAllowLocalNetworking = true;
```

Comment:
Is this some env var?

Reply:
It is a Nix-handled primitive derivation attribute: https://github.com/NixOS/nix/blob/f878b422b0b245d62593a909173b10b2b0588eaf/src/libstore/build/local-derivation-goal.cc#L2110; there are other uses in the Nixpkgs tree already.

Sorry for dropping the ball on this PR; I have been busy with moving lately and will endeavour to get back to it (and my other Nix work) in the next few weeks.